src/redux/books/BooksList.js

```diff
@@ -0,0 +1,41 @@
+// BooksList.js
```

Comment:
- [ ] @mrjuda As stated early kindly also move this file to your components directory.